FetchMigration/python/fetch_orchestrator.py

```diff
@@ -103,8 +104,12 @@ def run(params: FetchOrchestratorParams) -> Optional[int]:
         migration_monitor_params = MigrationMonitorParams(metadata_migration_result.target_doc_count,
                                                           params.get_local_endpoint())
         logging.info("Starting migration monitor...\n")
-        return migration_monitor.run(migration_monitor_params, proc)
+        return_code = migration_monitor.run(migration_monitor_params, proc)
+        # Suppress non-zero return code for graceful termination (SIGTERM)
+        if return_code == 143:
```

Comment:
A citation to the DataPrepper code (github) showing where/why it returns 143 would be great.  143 seems like it could be a fragile number to pin on.  This should also be a constant to document it a bit more.
I guess DataPrepper can also just run in "one-off" mode where it exits after a task is done (and then exits with code 0)?

Reply:
`143` is the [standard exit code](https://itsfoss.com/linux-exit-codes/) for a `SIGTERM` signal to indicate a graceful shutdown of the process. This behavior isn't managed by Data Prepper but rather [by the JVM](http://journal.thobe.org/2013/02/jvms-and-kill-signals.html).

I wasn't able to find any system libraries in Python that have this declared as a constant. IMO a test case isn't needed for this behavior since it is the expected behavior for any Linux process. Let me know if you disagree?